prow/plugins/trigger/ic_test.go

```diff
@@ -174,14 +282,14 @@ func TestHandleIssueComment(t *testing.T) {
 					Name:      "job",
 					AlwaysRun: true,
 					Context:   "pull-job",
-					Trigger:   "@k8s-bot test this",
+					Trigger:   "(^@k8s-bot test this\r?$|^/test\r?$)",
```

Comment:
Having the `|` here just complicates the test without providing any useful coverage of the non-test-code